manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -184,3 +184,26 @@ spec:
       for: 3m
       labels:
         severity: critical
+    - alert: etcdDivergentRevisions
+      annotations:
+        description: etcd is reporting divergent member revisions, with a pod
+          {{ $labels.pod }} on instance {{ $labels.instance }} trailing
+          {{ $value }} compacted revisions behind pod {{ $labels.max_pod }} on
+          instance {{ $labels.max_instance }}. Members may be stuck or, in
+          extreme circumstances, split brained.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdDivergentRevisions.md
+        summary: etcd is reporting divergent member revisions.
+      expr: |
+        -etcd_debugging_mvcc_compact_revision{job=~".*etcd.*"}
```

Comment:
Why is this compacted revision?

Reply:
From the commit message:

> * In healthy etcd clusters, the members seem to compact every 5 minutes.  There's some spread in Prometheus between the exact time when each compaction round happens.  Perhaps this is just random Prom-scrape ordering.  After 20 or 30s (approximately one default Prom scrape period), the reported compact revisions realign, and divergence within the etcd cluster drops back to zero.

For `etcd_debugging_mvcc_current_revision`, the Prom-scrape-induced latency means there's always some divergence, and you'd need to pick a threshold that depends on revision-velocity which is not as clean as the compact `> 0` threshold.